Code/GraphMol/Descriptors/Lipinski.cpp

```diff
@@ -469,5 +472,30 @@ unsigned int calcNumBridgeheadAtoms(const ROMol &mol,
   return atoms->size();
 }
 
+const std::string NumAtomStereoCentersVersion = "1.0.0";
+unsigned int numAtomStereoCenters(const ROMol &mol) {
```

Comment:
Ah. Good point. That rules that out.

At the very least the code should generate an error, but since people tend to ignore warnings and we know that the values will be wrong, I think it would be reasonable to throw a ValueErrorException